Web/View/ParagraphQuotes/Form.hs

```diff
@@ -4,15 +4,16 @@ import Web.View.Prelude
 import Web.Element.Types
 import Web.Element.ElementWrap
 
--- @todo: Add `required={isImageRequired}` to form.
 renderForm :: ParagraphQuote -> Bool -> Html
 renderForm paragraphQuote isImageRequired = formFor paragraphQuote [hsx|
     {(hiddenField #landingPageId)}
     {(hiddenField #weight)}
-    {visibleForm paragraphQuote}
+    {visibleForm paragraphQuote isImageRequired imageClasses}
     |]
     where
-            visibleForm paragraphQuote =
+            imageClasses = classes [("w-20 h-20", isJust paragraphQuote.imageUrl)]
+
+            visibleForm paragraphQuote isImageRequired imageClasses =
```

Comment:
If we don't pass the `isImageRequired` and `imageClasses` explicitly, we get errors. For example, if committing `imageClasses` we'll get

![image](https://github.com/amitaibu/ihp-landing-page/assets/125707/5999ea8b-b232-4716-958e-bb8472702c8f)

Why is that?